static/js/biomedical/disease/drug_table.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+import { GRAPH_BROWSER_REDIRECT } from "../bio_charts_utils";
+import { CompoundDiseaseTreatmentData } from "./chart";
+export interface DrugTableProps {
+  data: CompoundDiseaseTreatmentData[];
+}
+export function DrugTable(props: DrugTableProps): JSX.Element {
+  let DrugTableData = props.data;
```

Comment:
- DrugTableData -> drugTableData 
- can make this a constant and just do `const drugTableData = props.data.slice(0, 10)` because slice will just slice to the end of the array if the end idx is greater than the length of the array. And then you can just do `drugTableData.sort(...)` because sort mutates the array

Reply:
Done!